src/ipc-handlers.ts

```diff
@@ -665,18 +665,24 @@ export function openTerminalAtPath(
 			}
 		} else if ( platform === 'darwin' ) {
 			// macOS
-			if ( wpCliEnabled ) {
-				const script = `
+			const loadWpCliCommand =
+				'clear && export PATH=\\"${ cliPath }\\":$PATH && export STUDIO_APP_PATH=\\"${ appPath }\\" &&';
+			const script = `
 			tell application "Terminal"
 				if not application "Terminal" is running then launch
-				do script "clear && export PATH=\\"${ cliPath }\\":$PATH && export STUDIO_APP_PATH=\\"${ appPath }\\" && cd ${ targetPath }"
+				do script "${ wpCliEnabled ? loadWpCliCommand : '' } cd ${ targetPath } && clear"
 				activate
 			end tell
+
+			delay 0.5 -- Small delay to ensure the terminal is ready
+
+			tell application "System Events"
+				tell process "Terminal"
+					keystroke "v" using command down -- Simulate Cmd+V to paste the clipboard
+				end tell
+			end tell
```

Comment:
This new code requires some extra permissions, which can be a bit annoying. Let me know what you think.
I'm ok running the copy and open terminal, and the user will need to paste it manually.

You can test it by running `npm run make && open out/Studio-darwin-arm64` and then double click on Studio app.
Here are all permissions screenshots:
<img width="372" alt="Screenshot 2024-09-26 at 01 43 44" src="https://github.com/user-attachments/assets/8c8dfcf9-7e46-4d25-91e4-21b02dee784a">
<img width="372" alt="Screenshot 2024-09-26 at 01 43 48" src="https://github.com/user-attachments/assets/e748ed8b-69c4-4564-aa75-8c2c8b40e5fa">
<img width="529" alt="Screenshot 2024-09-26 at 01 43 52" src="https://github.com/user-attachments/assets/190e495a-fd9c-425a-9a4c-f0e4f91675bd">
<img width="827" alt="Screenshot 2024-09-26 at 01 44 02" src="https://github.com/user-attachments/assets/8b3c095c-55c1-4075-8985-5b9228fcd2ef">

Reply:
The `"Studio.app" would like to control this computer` one is a bit scary. 😬 

I wonder if it makes sense to add this button at all if we can’t automatically populate the Terminal window. Even with the button copy, I don’t think it's obvious that the user will need to manually paste the command.

Reply:
Should we simplify it and have an 'Open in Terminal'  button there, similar to the one in Overview?

Reply:
I replaced the behaviour. Now it says "Open in terminal" after clicking it copies the command and shows a notification, and opens the terminal.


https://github.com/user-attachments/assets/eca7c115-9ab9-4345-bb43-9a45e3d43847


@fluiddot , I was not able to display a message in the terminal. I could do "echo -n 'command'", but the message would appear twice. So I opted to use the notification.

Reply:
> @fluiddot , I was not able to display a message in the terminal. I could do "echo -n 'command'", but the message would appear twice. So I opted to use the notification.

I expected that it wouldn't be trivial to notify the user directly in the terminal, thanks for trying out. Using the system notification is a good approach 👍 .